web-server-lib/web-server/safety-limits.rkt

```diff
@@ -65,6 +69,7 @@
 
 
 (define-safety-limits/private-submodule
+  max-concurrent positive-count/c 1000 #:unlimited +inf.0
```

Comment:
This limit may be too conservative, so it might be worth raising.  It's a hard balance, though:

* if there's no limit, then it's easy to DoS the server, 
* if the default limit is too low, we may be limiting performance on very fast servers
* if the default limit is too high, slow servers may be open to DoS

IME, once you have several thousands of threads running, performance tends to degrade so `1000` feels right. Arguably, people doing very high performance stuff w/ the web server should be aware of and tweaking `safety-limits` anyway. I'll have to take a look at what other servers' defaults are.

cc @LiberalArtist

Reply:
In the end, I bumped this to 10k. Most of the servers I looked at had no default limit (but they did have ways to enforce a limit), except for gunicorn which has a limit of 1k by default.